crates/aptos-telemetry/src/build_information.rs

```diff
@@ -32,11 +32,17 @@ macro_rules! collect_build_information {
         // Get access to shadow BUILD information
         shadow_rs::shadow!(build);
 
+        // Get Git metadata from environment variables set during build-time.
+        // This is applicable for docker based builds.
+        const GIT_COMMIT_HASH: Option<&str> = option_env!("GIT_COMMIT_HASH");
```

Comment:
let's rename this to `GIT_SHA` for consistency reasons with other parts of the codebase